starr.go

```diff
@@ -14,22 +14,17 @@
 // covers about 10% of those. You can retrieve things like movies, albums, series
 // and books. You can retrieve server status, authors, artists and items in queues.
 // You can also add new media to each application with this library.
-//
 package starr
 
 import (
-	"context"
-	"crypto/tls"
 	"fmt"
-	"io"
 	"net/http"
-	"strings"
 	"time"
 )
 
 // Defaults for New().
 const (
-	DefaultTimeout = 10 * time.Second
+	DefaultTimeout = 30 * time.Second
```

Comment:
good call

Reply:
I've learned that some requests just take longer than 10 seconds, especially on an overloaded system.